Apps/Inc/Tasks.h

```diff
@@ -7,6 +7,7 @@
 #include "os.h"
 #include "common.h"
 #include "CarState.h"
```

Comment:
Should probably get rid of this header being included at some point

Reply:
Fixed in #201